07-routing-gossip.md

```diff
@@ -289,6 +289,12 @@ The receiving node MAY use `rgb_color` and `alias` to reference nodes in interfa
 
 ### Rationale
 
+The `timestamp` field is used by nodes for pruning (either if it's too
+far in the future, or if it's been two weeks with no update), so it
+makes sense to have it be a UNIX timestamp (ie. seconds since UTC
+1970-01-01).  It need only be exact, however, given the unusual case
```

Comment:
> It need only be exact, however, given the unusual case of two `channel_update`s within a second.

I don't understand this, what do you mean?

Reply:
Same, can't really parse this section.

Reply:
Oops, should be "We can't require that it be a timestamp, however..."